src/frontend/Komponenter/Behandling/SettPåVent/TaAvVentModal.tsx

```diff
@@ -12,7 +12,7 @@ const AlertStripe = styled(Alert)`
 `;
 
 export const TaAvVentModal: FC<{ behandlingId: string }> = ({ behandlingId }) => {
-    const { taAvVentStatus, hentBehandling } = useBehandling();
+    const { taAvVentStatus, hentBehandling, settTaAvVentStatus } = useBehandling();
```

Comment:
Skulle man kunne flytte in taAvVentStatus-statet i `InfostripeSattPåVent` i stedet, og sende med taAvVentStatus og settAvVentStatus til modalen? Og ha modalen her i denne kompontenten i stedet? 

Då legger vi ikke inn for mye i behandlingcontexten, og holder statet mer nært der det vises

Reply:
Ser Eirik svarte på det her https://github.com/navikt/familie-ef-sak-frontend/pull/1951#discussion_r1071308897
Tenker man burde kunne legge staten i `InfostripeSattPåVent ` og ha modalen rett i den komponenten for å få det samlet :-)